.github/workflows/wheels-test.yaml

```diff
@@ -151,6 +144,13 @@ jobs:
         fetch-depth: 0 # unshallow fetch for setuptools-scm
         persist-credentials: false
 
+    - name: Telemetry setup
+      uses: rapidsai/shared-actions/telemetry-dispatch-setup@main
+      continue-on-error: true
+      if: ${{ vars.TELEMETRY_ENABLED == 'true' && github.run_attempt == '1' }}
+      with:
+        extra_attributes: "rapids.operation=test-wheel,rapids.package_type=wheel,rapids.cuda=${{ matrix.CUDA_VER }},rapids.py=${{ matrix.PY_VER }},rapids.arch=${{ matrix.ARCH }},rapids.linux=${{ matrix.LINUX_VER }},rapids.gpu=${{ matrix.GPU }},rapids.driver=${{ matrix.DRIVER }},rapids.deps=${{ matrix.DEPENDENCIES }}"
```

Comment:
I wish these keys matches exactly. Like “DEPENDENCIES” (or maybe “dependencies”) instead of “deps.”